assets/pages/bundles-landing/support-landing-ab-test/components/subscriptionBundleNewDesign.jsx

```diff
@@ -14,9 +17,10 @@ type PropTypes = {
   heading: string,
   price: string,
   from: boolean,
-  copy: string,
+  copy: ListItem[],
```

Comment:
Maybe not for this PR, but I find it very confusing when we use the word "copy" to mean "text supplied by editorial".  Maybe in this context `description` might be clearer?

Reply:
I understand, I can change it to 'benefits' if that helps? Description puts me more in an a11y mindset.

The word 'copy' is almost always used by design and UX when talking about this type of content, so it's become my word for it too 🤓.

Reply:
Benefits is fairly clear to me.  You should stop hanging out with design & UX, they are clearly a bad influence.